connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -308,12 +319,12 @@ export async function syncZendeskCategoryActivity({
  */
 export async function syncZendeskArticlesActivity({
   connectorId,
-  categoryId,
+  category,
```

Comment:
Can some1 confirm that this does not causes a risk of blowing up Temporal's event history? 🙏

Reply:
We definitely prefer short ids to actual strings to limit state but more importantly to avoid leaking readable user info to temporal

Reply:
missed that one, will fix in a follow up PR